db/schema.rb

```diff
@@ -375,11 +376,19 @@
     t.index ["id"], name: "index_sequences_on_id"
   end
 
-  create_table "writers", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
I don't remember where this writers table came from or know what it was for. I think it got into a development db at some point and snuck into the schema. There's certainly not a migration for it. I got rid of it.